xml/System.Reflection/TypeInfo.xml

```diff
@@ -73,7 +73,7 @@
   
  A <xref:System.Reflection.TypeInfo> object represents the type definition itself, whereas a <xref:System.Type> object represents a reference to the type definition. Getting a <xref:System.Reflection.TypeInfo> object forces the assembly that contains that type to load. In comparison, you can manipulate <xref:System.Type> objects without necessarily requiring the runtime to load the assembly they reference.  
   
- In the [!INCLUDE[net_win8_profile](~/includes/net-win8-profile-md.md)], you use the reflection properties of <xref:System.Reflection.TypeInfo> that return <xref:System.Collections.Generic.IEnumerable%601> collections instead of methods that return arrays. For example, use the <xref:System.Reflection.TypeInfo.DeclaredMembers%2A> property to get all declared members, or the <xref:System.Reflection.TypeInfo.DeclaredProperties%2A> property to get all declared properties. Reflection contexts can implement lazy traversal of these collections for large assemblies or types. To get specific members, use methods such as <xref:System.Reflection.TypeInfo.GetDeclaredMethod%2A> and <xref:System.Reflection.TypeInfo.GetDeclaredProperty%2A>, and pass the name of the method or property you would like to retrieve.  
+ In the [!INCLUDE[net_win8_profile](~/includes/net-win8-profile-md.md)], you use the reflection properties of <xref:System.Reflection.TypeInfo> that return <xref:System.Collections.Generic.IEnumerable%601> collections instead of methods that return arrays. For example, use the <xref:System.Reflection.TypeInfo.DeclaredMembers%2A> property to get all declared members, or the <xref:System.Reflection.TypeInfo.%2A> property to get all declared properties. Reflection contexts can implement lazy traversal of these collections for large assemblies or types. To get specific members, use methods such as <xref:System.Reflection.TypeInfo.GetDeclaredMethod%2A> and <xref:System.Reflection.TypeInfo.GetDeclaredProperty%2A>, and pass the name of the method or property you would like to retrieve.  
```

Comment:
```suggestion
 In the [!INCLUDE[net_win8_profile](~/includes/net-win8-profile-md.md)], you use the reflection properties of <xref:System.Reflection.TypeInfo> that return <xref:System.Collections.Generic.IEnumerable%601> collections instead of methods that return arrays. For example, use the <xref:System.Reflection.TypeInfo.DeclaredMembers%2A> property to get all declared members, or the <xref:System.Reflection.TypeInfo.DeclaredProperties%2A> property to get all declared properties. Reflection contexts can implement lazy traversal of these collections for large assemblies or types. To get specific members, use methods such as <xref:System.Reflection.TypeInfo.GetDeclaredMethod%2A> and <xref:System.Reflection.TypeInfo.GetDeclaredProperty%2A>, and pass the name of the method or property you would like to retrieve.  
```

Reply:
Adds "DeclaredProperties" back to xref.